nansat/tests/test_exporter.py

```diff
@@ -264,28 +263,27 @@ def test_export2thredds_arctic_long_lat(self):
         self.assertTrue(os.path.exists(tmpfilename))
         g = gdal.Open(tmpfilename)
         metadata = g.GetMetadata_Dict()
-
-        # Test that the long/lat values are set aproximately correct
-        ncg = 'NC_GLOBAL#'
-        easternmost_longitude = metadata.get(ncg + 'easternmost_longitude')
-        self.assertTrue(float(easternmost_longitude) > 179,
-                        'easternmost_longitude is wrong:' +
-                        easternmost_longitude)
-        westernmost_longitude = metadata.get(ncg + 'westernmost_longitude')
-        self.assertTrue(float(westernmost_longitude) < -179,
-                        'westernmost_longitude is wrong:' +
-                        westernmost_longitude)
-        northernmost_latitude = metadata.get(ncg + 'northernmost_latitude')
-        self.assertTrue(float(northernmost_latitude) > 89.999,
-                        'northernmost_latitude is wrong:' +
-                        northernmost_latitude)
-        southernmost_latitude = metadata.get(ncg + 'southernmost_latitude')
-        self.assertTrue(float(southernmost_latitude) < 54,
-                        'southernmost_latitude is wrong:' +
-                        southernmost_latitude)
-        self.assertTrue(float(southernmost_latitude) > 53,
-                        'southernmost_latitude is wrong:' +
-                        southernmost_latitude)
+        pprint(metadata)
+
+        # Test that the long/lat values are set correctly
+		# GDAL behaves differently:
+		# Windows: nc-attributes are accessible without 'NC_GLOBAL#' prefix
+		# Linux: nc-attributes are accessible only with 'NC_GLOBAL#' prefix
+        if 'linux' in sys.platform:
```

Comment:
Are this changes necessary only for the passing of the test, and if so,  will this work ok with windows in real life? Could you educate me a little bit or point to the code, please, because I do not clearly understand that?

Reply:
That's needed only for passing the tests. I use GDAL directly to test the files independently (not with nansat). And it turns that it works differently on Win and Linux. On Win it read all metadata and doesn't add NC_GLOBAL, on Linux it shows only global metadata and adds 'NCGLOBAL' in front.

Reply:
What about osx? I think you should default to nc_prefix = 'NC_GLOBAL#' unless it's windows - or possible better as I fixed this earlier, I checked if a metadata record existed with 'NC_GLOBAL#'

https://github.com/nansencenter/nansat/commit/172d31550bff4f5ef3ae7e10f893b13c86889fd3#diff-60a6bd1b9e79d00812bde1d1b98aba33

Reply:
Good approach! I'm quite surprised it was not in the current branch...

Reply:
Ahh, it probably disappeared when we were refactoring the code. Then we have split these tests from nansat_tests into exporter_tests.py